.github/workflows/uploadFirrtlReleaseArtifacts.yml

```diff
@@ -85,57 +99,53 @@ jobs:
 
       - name: Build LLVM
         run: |
+          ${{ matrix.setup }}
           mkdir -p llvm/build
           cd llvm/build
-          cmake -G Ninja ../llvm \
-              ${{ matrix.build_config.cmake-args }} \
-              -DCMAKE_BUILD_TYPE=${{ matrix.build_config.mode }} \
-              -DCMAKE_C_COMPILER=${{ matrix.cc }} \
-              -DCMAKE_CXX_COMPILER=${{ matrix.cxx }} \
-              -DBUILD_SHARED_LIBS=${{ matrix.build_config.shared }} \
-              -DLLVM_BUILD_TOOLS=OFF \
-              -DLLVM_BUILD_EXAMPLES=OFF \
-              -DLLVM_ENABLE_ASSERTIONS=${{ matrix.build_config.assert }} \
-              -DLLVM_ENABLE_BINDINGS=OFF \
-              -DLLVM_ENABLE_OCAMLDOC=OFF \
-              -DLLVM_ENABLE_PROJECTS='mlir' \
-              -DLLVM_OPTIMIZED_TABLEGEN=ON \
-              -DLLVM_STATIC_LINK_CXX_STDLIB=ON \
-              -DLLVM_ENABLE_TERMINFO=OFF \
-              -DLLVM_PARALLEL_LINK_JOBS=1 \
-              -DLLVM_TARGETS_TO_BUILD="host" \
-              -DLLVM_FORCE_ENABLE_STATS=${{ matrix.build_config.stats }} \
+          cmake -G Ninja ../llvm ${{ matrix.cont }}
+              ${{ matrix.cmake-args }} ${{ matrix.cont }}
+              -DCMAKE_BUILD_TYPE=${{ matrix.build_config.mode }} ${{ matrix.cont }}
+              -DBUILD_SHARED_LIBS=${{ matrix.build_config.shared }} ${{ matrix.cont }}
+              -DLLVM_BUILD_TOOLS=OFF ${{ matrix.cont }}
+              -DLLVM_BUILD_EXAMPLES=OFF ${{ matrix.cont }}
+              -DLLVM_ENABLE_ASSERTIONS=${{ matrix.build_config.assert }} ${{ matrix.cont }}
+              -DLLVM_ENABLE_BINDINGS=OFF ${{ matrix.cont }}
+              -DLLVM_ENABLE_OCAMLDOC=OFF ${{ matrix.cont }}
+              -DLLVM_ENABLE_PROJECTS='mlir' ${{ matrix.cont }}
+              -DLLVM_OPTIMIZED_TABLEGEN=ON ${{ matrix.cont }}
+              -DLLVM_STATIC_LINK_CXX_STDLIB=ON ${{ matrix.cont }}
+              -DLLVM_ENABLE_TERMINFO=OFF ${{ matrix.cont }}
+              -DLLVM_PARALLEL_LINK_JOBS=1 ${{ matrix.cont }}
+              -DLLVM_TARGETS_TO_BUILD="host" ${{ matrix.cont }}
+              -DLLVM_FORCE_ENABLE_STATS=${{ matrix.build_config.stats }} ${{ matrix.cont }}
               -DLLVM_ENABLE_ZSTD=OFF
           ninja
-          #Checks temporarily disabled because current LLVM commit (9305b63d6) fails checks
-          #ninja check-llvm check-mlir
```

Comment:
Deleted because they were commented out anyway and they still don't work. @dtzWill may have thoughts.

Reply:
I've added `ninja check-mlir`. I've spun up a couple of runs.

I think if we're going to test mlir here we really need to test it elsewhere because finding out that `check-mlir` is broken after tagging a release is not ideal (but I grant we should check it here as well).